src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -148,6 +148,9 @@ public byte[] mask(byte[] input) {
                 maskStringValueInPlace(maskingState);
             }
         }
+
+        FixedLengthTargetValueMaskUtil.flushReplacementOperations(maskingState);
```

Comment:
I think we should rename the class `FixedLengthTargetValueMaskUtil`. I have no suggestions 😅 😄

Reply:
Renamed to `ValueMaskingUtil`, adjusted docs. I think whole "fixed length" in methods and class name doesn't make much sense as we use it as another parameter. I was initially quite confused what "fixed length" even means when I was looking at this method from `KeyContainsMasker` (that's why I renamed it to `maskLength`), but either "fixed length" or "mask length" are quite obvious and IMO it's enough to only refer to them as arguments, without duplicating in class names or method names